### PR TITLE
Tweak status messages for data platform providers

### PR DESCRIPTION
**Pull request by reefdog**
_This PR ladders off #196._

Wrap status messages in the `quiet` styling and tweak the wording.

### CODE REVIEW

File: src/components/DataPlatformProvider/DataPlatformProviderLoader.tsx [hunk 1]
```diff
@@ -138,7 +138,9 @@ const Invalid = ({
       </PanelTitle>
     )}
   >
-    {`The data platform provider ${provider} is not recognized.`}
+    <div className="quiet">
+      {`The data platform provider “${provider}” is not recognized.`}
```
Comment (jasonaowen) [inline src/components/DataPlatformProvider/DataPlatformProviderLoader.tsx:142]: Curly quotes? <shakes head> and I thought I knew you...~

File: src/components/DataPlatformProvider/DataPlatformProviderLoader.tsx [hunk 2]
```diff
@@ -174,7 +178,9 @@ const NoData = ({
       </PanelTitle>
     )}
   >
-    No data found
+    <div className="quiet">
+      {`No ${providers[provider].name} data found.`}
```
Comment (jasonaowen) [inline src/components/DataPlatformProvider/DataPlatformProviderLoader.tsx:182]: I worry a little bit about repetition with the title above!

**Review by jasonaowen — APPROVED** [review]
Thank you for cleaning up my placeholder copy, @reefdog, in both style and substance!

File: src/components/DataPlatformProvider/DataPlatformProviderLoader.tsx (hunk 2 above)
Comment (reefdog) [reply src/components/DataPlatformProvider/DataPlatformProviderLoader.tsx:182]: Actually… there's a very good point that I didn't even look at the titles! I'm not sure I love using it for status text ("Invalid Provider") or for the provider name during loading. It feels a little strange to me, but I want to muse on it for a few more minutes.

Regardless, I don't mind the repetition so much, and like the explicitness of the provider name in the message, unless you hard disagree!

File: src/components/DataPlatformProvider/DataPlatformProviderLoader.tsx (hunk 1 above)
Comment (reefdog) [reply src/components/DataPlatformProvider/DataPlatformProviderLoader.tsx:142]: 😱 Curly quotes forever! (I am as much of a typographical snob as I can afford to be.)

File: src/components/DataPlatformProvider/DataPlatformProviderLoader.tsx (hunk 2 above)
Comment (reefdog) [reply src/components/DataPlatformProvider/DataPlatformProviderLoader.tsx]: I came down on "you're right"!

**Pull request merged**